docs/service-mesh/03-02-istio-patch.md

```diff
@@ -5,14 +5,12 @@ type: Details
 
 As a core component, Istio is installed with every Kyma deployment by default. The installation includes the following steps:
 
-1. Istio is installed using the official charts from the currently supported release. The charts reside in the `resources/istio` directory.
+1. Istio is installed using the `istioctl` tool. The installation resources reside in the `resources/istio` directory.
```

Comment:
```suggestion
1. Istio is installed using the `istioctl` tool. The installation resources reside in the `resources/istio` directory.
```

Reply:
There are no changes in this suggestion 😕